README.md

```diff
@@ -235,7 +235,7 @@ By the way, Gorgonia comes with nice-ish graph printing abilities. Here's an exa
 
 ![graph1](https://raw.githubusercontent.com/chewxy/gorgonia/master/media/exprGraph_example2.png)
 
-To read the graph is easy. The expression builds from bottom up, while the derivations build from top down. This way the derivative of each node is roughly on the same level. 
+To read the graph is easy. The expression builds from bottom up, while the derivations build from the top down. This way the derivative of each node is roughly on the same level. 
```

Comment:
Extraneous "the". The sentence follows "...builds from bottom up ... builds from top down"